### PR TITLE
Add medium link to Docusaurs page footer

### LINKED ISSUE #171: Add link with icon to our AnitaB.org Open Source Medium
## Description

As an admin,
I need our Medium blog to be in the social media links on Docusaurus,
so that I can promote our blog to our contributors.

We want to see the medium link to our AnitaB.org Open Source on the Docusaurs page footer.
Our Medium publication: https://medium.com/anitab-org-open-source
Our Docusaurus documentation: https://anitab-org.github.io/anitab-forms-web/

## Mocks

You can find this in the footer:
docusaurus page footer

### Update [Required]
- [ ] Add Medium icon with link to our blog on Docusaurus page footer.

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation

30 min

### PR DESCRIPTION
**Pull request by ToluAkin**
### Description

- Add medium link to Docusaurs page footer

Fixes #171

### Type of Change:
- Add data object to the file `docusaurus.config.js`

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Run the code

### Checklist:
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by ToluAkin** [comment]
@isabelcosta

**Comment by codesankalp** [comment]
@ToluAkin Please update the PR description also.

**Comment by isabelcosta** [comment]
@ToluAkin I will review ;) Just as FYI, I changed your issue description to have the link to the issue you are solving. Usually, you have to add like `Fixes #123` where 123 is the number of the issue. In this way, once this is merged GitHub will close the issue automatically.

File: docs/docusaurus.config.js
Comment (codesankalp) [inline docs/docusaurus.config.js]: I think both are the same, so remove one.

**Review by codesankalp — CHANGES_REQUESTED** [review]

File: docs/docusaurus.config.js
Comment (isabelcosta) [reply docs/docusaurus.config.js]: Good point! I only noticed that now. thank you @codesankalp 🙌🏾

File: docs/docusaurus.config.js
Comment (isabelcosta) [inline docs/docusaurus.config.js]: you can remove this too :) Having it under `Zulip` on the `Community` column is enough :)

**Review by isabelcosta — CHANGES_REQUESTED** [review]
Thank you for contributing. Once you address my comment and @codesankalp 's comment I am happy to approve :)

**Review by isabelcosta — DISMISSED** [review]
Thank you so much for your contribution @ToluAkin ! This looks great!

**Comment by ToluAkin** [comment]
@isabelcosta All checks are passing now. Is the PR title okay? Do I need to remove the phrase after the issue number?

**Review by isabelcosta — APPROVED** [review]
Thank you for fixing the failing check! Even though you did not need it 🤗 
I'll approve! Thank you for contributing @ToluAkin !

**Comment by isabelcosta** [comment]
@ToluAkin the PR title looks good. I wouldn't add the "Fixes #171" part, since it is already in the issue description. But its fine, as long as the title also indicates what the PR is actually fixing :) So it looks great to me!

**Comment by ToluAkin** [comment]
> @ToluAkin the PR title looks good. I wouldn't add the "Fixes #171" part, since it is already in the issue description. But its fine, as long as the title also indicates what the PR is actually fixing :) So it looks great to me!

Thank you!

**Comment by ToluAkin** [comment]
Hi @codesankalp , @vj-codes Could you help me check this again if it is good to go? Thanks.

**Review by Aaishpra — APPROVED** [review]
Great work Thanks for contributing @ToluAkin

**Comment by Aaishpra** [comment]
I have also added the Hacktoberfest label here.

**Comment by ToluAkin** [comment]
> I have also added the Hacktoberfest label here.

😃 ..... Thank you! I'm happy to help.

**Review by vj-codes — APPROVED** [review]
LGTM 🎉

**Comment by isabelcosta** [comment]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

   * _Run docusaurus website and see footer_  
     _Screenshot/gif_:  image
     _Expected Result_:  Medium publication link should appear in the footer of Docusaurus
     _Actual Result_: as expected there is a medium link in the community tab
  
3. Additional Comments: steps to run this (also following [our QA guide](https://github.com/anitab-org/documentation/blob/master/quality-assurance.md))
    ```
    cd docs
    yarn install
    yarn start
    go to http://localhost:3000/anitab-forms-web/
    ```
4. Status of PR Changed to: Ready to Merge.

**Comment by isabelcosta** [comment]
@ToluAkin I would suggest you to in the "How Has This Been Tested?" next time to show a screenshot showing you tested the change. That makes for a good pull request description that can help maintainers look for your change :)

**Pull request merged**
